test/manipulation.test.js

```diff
@@ -743,7 +743,15 @@ describe('manipulation', function() {
       Todo = db.define('Todo', {
         content: String,
       });
-      db.automigrate(['Post', 'Todo'], done);
+      Person = db.define('Person', {
+        name: String,
+        gender: String,
+        married: Boolean,
+        age: {type: Number, index: true},
+        dob: Date,
+        createdAt: {type: Date, default: Date},
```

Comment:
On the second look, the Person model does not seem to be used at all, can you remove it please?